tests/test_openjdk.py

```diff
@@ -2,6 +2,34 @@
 import pytest
 from bci_tester.data import OPENJDK_11_CONTAINER
 from bci_tester.data import OPENJDK_17_CONTAINER
+from pytest_container import DerivedContainer
+from pytest_container.container import container_from_pytest_param
+from pytest_container import OciRuntimeBase
+
+condir = "/tmp/"
+appdir = "tests/trainers/java/"
+
+DOCKERF_EXTENDED = f"""
+WORKDIR {condir}
+COPY {appdir} {condir}
+"""
+
+CONTAINER_IMAGES = [
+    OPENJDK_11_CONTAINER,
+    OPENJDK_17_CONTAINER,
+]
+
+CONTAINER_IMAGES_EXTENDED = [
+    pytest.param(
+        DerivedContainer(
+            base=container_from_pytest_param(CONTAINER_T),
+            containerfile=DOCKERF_EXTENDED,
+        ),
+        marks=CONTAINER_T.marks,
+        id=CONTAINER_T.id,
+    )
+    for CONTAINER_T in CONTAINER_IMAGES
```

Comment:
Usually only constants use all-caps
```suggestion
    pytest.param(
        DerivedContainer(
            base=container_from_pytest_param(container),
            containerfile=DOCKERF_EXTENDED,
        ),
        marks=container.marks,
        id=container.id,
    )
    for container in CONTAINER_IMAGES
```

Reply:
yes, I copy this from somewhere

Reply:
done